travis/run_pylint.py

```diff
@@ -9,6 +9,8 @@
 import click
 import pylint.lint
 
+import getaddons
+
```

Comment:
@moylop260 I don't understand your comment. Here I'm just adding a filter to an existing method. Do you want me to do some refactoring here?